### PR TITLE
Bug fix/cd and wrong path

### PR DESCRIPTION
**Pull request by fceller**

### CODE REVIEW

File: helper.fish [hunk 1]
```diff
@@ -129,9 +129,10 @@ function checkoutIfNeeded
 end
 
 function clearResults
-  cd $WORKDIR/work
-  for f in testreport* ; rm -f $f ; end
-  rm -f test.log buildArangoDB.log cmakeArangoDB.log
+  pushd $WORKDIR/work
+  and for f in testreport* ; rm -f $f ; end
+  and rm -f test.log buildArangoDB.log cmakeArangoDB.log
+  and popd
```
Comment (KVS85) [inline helper.fish]: Won't `popd` be better without `and`?

File: helper.linux.fish [hunk 2]
```diff
@@ -28,58 +28,70 @@ function compiler
...
-  cd $WORKDIR/containers/buildUbuntu.docker
-  docker build -t $UBUNTUBUILDIMAGE .
-  rm -f $WORKDIR/containers/buildUbuntu.docker/scripts/*.fish
-  cd $WORKDIR
+  pushd $WORKDIR
+  and cp -a scripts/{makeArangoDB,buildArangoDB,checkoutArangoDB,checkoutEnterprise,clearWorkDir,downloadStarter,downloadSyncer,runTests,runFullTests,switchBranches,recursiveChown}.fish containers/buildUbuntu.docker/scripts
+  and cd $WORKDIR/containers/buildUbuntu.docker
+  and docker build -t $UBUNTUBUILDIMAGE .
+  and rm -f $WORKDIR/containers/buildUbuntu.docker/scripts/*.fish
+  and popd
```
Comment (KVS85) [inline helper.linux.fish]: Won't `popd` be better without `and`?

File: helper.linux.fish [hunk 3]
```diff
@@ -28,58 +28,70 @@ function compiler
...
-  cd $WORKDIR/containers/buildUbuntuPackaging.docker
-  docker build -t $UBUNTUPACKAGINGIMAGE .
-  rm -f $WORKDIR/containers/buildUbuntuPackaging.docker/scripts/*.fish
-  cd $WORKDIR
+  pushd $WORKDIR
+  and cp -a scripts/buildDebianPackage.fish containers/buildUbuntuPackaging.docker/scripts
+  and cd $WORKDIR/containers/buildUbuntuPackaging.docker
+  and docker build -t $UBUNTUPACKAGINGIMAGE .
+  and rm -f $WORKDIR/containers/buildUbuntuPackaging.docker/scripts/*.fish
+  and popd
```
Comment (KVS85) [inline helper.linux.fish]: Won't `popd` be better without `and`?

File: helper.linux.fish [hunk 4]
```diff
@@ -28,58 +28,70 @@ function compiler
...
-  cd $WORKDIR/containers/buildAlpine.docker
-  docker build -t $ALPINEBUILDIMAGE .
-  rm -f $WORKDIR/containers/buildAlpine.docker/scripts/*.fish
-  cd $WORKDIR
+  pushd $WORKDIR
+  and cp -a scripts/makeAlpine.fish scripts/buildAlpine.fish containers/buildAlpine.docker/scripts
+  and cd $WORKDIR/containers/buildAlpine.docker
+  and docker build -t $ALPINEBUILDIMAGE .
+  and rm -f $WORKDIR/containers/buildAlpine.docker/scripts/*.fish
+  and popd
```
Comment (KVS85) [inline helper.linux.fish]: Won't `popd` be better without `and`?

File: helper.linux.fish [hunk 5]
```diff
@@ -28,58 +28,70 @@ function compiler
...
-  cd $WORKDIR/containers/buildCentos7Packaging.docker
-  docker build -t $CENTOSPACKAGINGIMAGE .
-  rm -f $WORKDIR/containers/buildCentos7Packaging.docker/scripts/*.fish
-  cd $WORKDIR
+  pushd $WORKDIR
+  and cp -a scripts/buildRPMPackage.fish containers/buildCentos7Packaging.docker/scripts
+  and cd $WORKDIR/containers/buildCentos7Packaging.docker
+  and docker build -t $CENTOSPACKAGINGIMAGE .
+  and rm -f $WORKDIR/containers/buildCentos7Packaging.docker/scripts/*.fish
+  and popd
```
Comment (KVS85) [inline helper.linux.fish]: Won't `popd` be better without `and`?

File: helper.linux.fish [hunk 6]
```diff
@@ -348,28 +376,30 @@ function oskarLimited
...
+  and git push
+  and buildUbuntuBuildImage
+  and pushUbuntuBuildImage
+  and buildAlpineBuildImage
+  and pushAlpineBuildImage
+  and buildUbuntuPackagingImage
+  and pushUbuntuPackagingImage
+  and buildCentosPackagingImage
+  and pushCentosPackagingImage
+  and popd
```
Comment (KVS85) [inline helper.linux.fish]: Won't `popd` be better without `and`?

File: helper.linux.fish [hunk 7]
```diff
@@ -348,28 +376,30 @@ function oskarLimited
...
-  cd $WORKDIR
+  pushd $WORKDIR
   and git checkout -- .
   and git pull
   and source helper.fish
   and pullUbuntuBuildImage
   and pullAlpineBuildImage
   and pullUbuntuPackagingImage
   and pullCentosPackagingImage
+  and popd
```
Comment (KVS85) [inline helper.linux.fish]: Won't `popd` be better without `and`?

File: helper.linux.fish [hunk 8]
```diff
@@ -393,15 +423,17 @@ function makeDockerImage
...
     echo Could not create install tarball!
+    popd
     return 1
   end
 
-  cd $WORKDIR/containers/arangodb.docker
-  docker build -t $imagename .
+  pushd $WORKDIR/containers/arangodb.docker
+  and docker build -t $imagename .
+  and popd
```
Comment (KVS85) [inline helper.linux.fish]: Won't `popd` be better without `and`?

File: helper.fish (hunk 1 above)
Comment (fceller) [reply helper.fish]: done

File: helper.linux.fish (hunk 2 above)
Comment (fceller) [reply helper.linux.fish]: done

File: helper.linux.fish (hunk 3 above)
Comment (fceller) [reply helper.linux.fish]: done

File: helper.linux.fish (hunk 4 above)
Comment (fceller) [reply helper.linux.fish]: done

File: helper.linux.fish (hunk 5 above)
Comment (fceller) [reply helper.linux.fish]: done

File: helper.linux.fish (hunk 6 above)
Comment (fceller) [reply helper.linux.fish]: done

File: helper.linux.fish (hunk 7 above)
Comment (fceller) [reply helper.linux.fish]: done

File: helper.linux.fish (hunk 8 above)
Comment (fceller) [reply helper.linux.fish]: done

**Pull request merged**